<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_main</class>
 <widget class="QWidget" name="Form_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/BJ1.png);</string>
   </property>
   <widget class="QPushButton" name="pushButtonduidian">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>412</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{color: rgb(0, 0, 0);font: 90 22pt &quot;Aharoni&quot;;  }
QPushButton{border-image: url(:/twhite.png);   }
QPushButton:hover{border-image: url(:/K7.png);color: rgb(255, 255, 255);}  
QPushButton:pressed{border-image: url(:/K7.png);color: rgb(255, 255, 255); }

</string>
    </property>
    <property name="text">
     <string>对点广播</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>412</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{color: rgb(0, 0, 0);font: 90 22pt &quot;Aharoni&quot;;}
QPushButton{border-image: url(:/twhite.png);}
QPushButton:hover{border-image: url(:/K7.png);color: rgb(255, 255, 255);}  
QPushButton:pressed{border-image: url(:/K7.png);color: rgb(255, 255, 255);}
</string>
    </property>
    <property name="text">
     <string>分区广播</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>412</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{color: rgb(0, 0, 0);font: 90 22pt &quot;Aharoni&quot;;}
QPushButton{border-image: url(:/twhite.png);}
QPushButton:hover{border-image: url(:/K7.png);color: rgb(255, 255, 255);}  
QPushButton:pressed{border-image: url(:/K7.png);color: rgb(255, 255, 255);}
</string>
    </property>
    <property name="text">
     <string>资源点播</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Aharoni&quot;;
color: rgb(255, 255, 255);
border-image: url(:/B12.png);</string>
    </property>
    <property name="text">
     <string>菱声广播系统呼叫</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Aharoni&quot;;
color: rgb(255, 255, 255);
border-image: url(:/B12.png);</string>
    </property>
    <property name="text">
     <string>编号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;Aharoni&quot;;
border-image: url(:/B12.png);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>02</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>801</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/BKh.png);</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>80</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{color: rgb(0, 0, 0);font: 90 22pt &quot;Aharoni&quot;; border-radius:10px;}
QPushButton{border-image: url(:/twhite.png);}
QPushButton:hover{border-image: url(:/K7.png);color: rgb(255, 255, 255); border-radius:10px;}  
QPushButton:pressed{border-image: url(:/K7.png);color: rgb(255, 255, 255); border-radius:10px;}
</string>
      </property>
      <property name="text">
       <string>全选</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>661</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{font: 75 12pt &quot;Aharoni&quot;;color: rgb(255, 255, 255);border-image: url(:B12.png); margin-top:20px; margin-bottom:20px;margin-left:20px;margin-right:20px;}

QListWidget::Item{ margin-top:10px; margin-bottom:10px;margin-left:15px;margin-right:15px;}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>230</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{color: rgb(255, 255, 255);font: 90 30pt &quot;Aharoni&quot;;  border-radius:10px;}
QPushButton{border-image: url(://KR.png);  border-radius:10px;}
QPushButton:hover{border-image: url(:/K6.png);  border-radius:10px;}  
QPushButton:pressed{border-image: url(:/K6.png);  border-radius:10px;}</string>
      </property>
      <property name="text">
       <string>呼叫</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>150</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{color: rgb(0, 0, 0);font: 90 22pt &quot;Aharoni&quot;; border-radius:10px;}
QPushButton{border-image: url(:/twhite.png);}
QPushButton:hover{border-image: url(:/K7.png);color: rgb(255, 255, 255); border-radius:10px;}  
QPushButton:pressed{border-image: url(:/K7.png);color: rgb(255, 255, 255); border-radius:10px;}
</string>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>230</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{color: rgb(255, 255, 255);font: 90 30pt &quot;Aharoni&quot;;  border-radius:10px;}
QPushButton{border-image: url(://KR.png);  border-radius:10px;}
QPushButton:hover{border-image: url(:/K6.png);  border-radius:10px;}  
QPushButton:pressed{border-image: url(:/K6.png);  border-radius:10px;}</string>
      </property>
      <property name="text">
       <string>呼叫</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{color: rgb(0, 0, 0);font: 90 22pt &quot;Aharoni&quot;; border-radius:10px;}
QPushButton{border-image: url(:/twhite.png);}
QPushButton:hover{border-image: url(:/K7.png);color: rgb(255, 255, 255); border-radius:10px;}  
QPushButton:pressed{border-image: url(:/K7.png);color: rgb(255, 255, 255); border-radius:10px;}
</string>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>641</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget{font: 75 12pt &quot;Aharoni&quot;;color: rgb(255, 255, 255);border-image: url(:B12.png); margin-top:20px; margin-bottom:20px;margin-left:20px;margin-right:20px;}

QListWidget::Item{ margin-top:10px; margin-bottom:10px;margin-left:15px;margin-right:15px;}</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QListWidget" name="listWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>631</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 26pt &quot;Aharoni&quot;;
color: rgb(255, 255, 255);
border-image: url(:/B12.png);</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
